src/HW_models/nrfx_bsim_redef.h

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (c) 2018 Oticon A/S
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * This file redefines macros from nrfx that are not to be used
```

Comment:
Maybe better:
"This file redefines macros from nrfx that need to be different for simulated devices." ?  (note the typo too)